src/frontend/components/UI/TextInputField/index.tsx

```diff
@@ -4,7 +4,7 @@ import ContextProvider from 'frontend/state/ContextProvider'
 import './index.css'
 
 interface TextInputFieldProps {
-  htmlId: string
+  htmlId?: string
```

Comment:
we should always have an html id so we can pair the input with the corresponding label properly, do you need this change for anything specific?

Reply:
I saw that some uses of `TextInputField` (like the one in Legendary's save sync page) *only* specified a htmlId & not a label. That seemed redundant to me (without a label, there's nothing to link)

I should've used union types here (so that if you specify either htmlId or label, you also have to specify the other

Reply:
Testing this further, it seems that React does not like me using Unions here

Returning this to how it was before then